package.json

```diff
@@ -56,6 +56,7 @@
     "@types/shortid": "^0.0.29",
     "applicationinsights": "^1.2.0",
     "body-parser": "^1.18.3",
+    "bootstrap-datepicker": "1.8.0",
```

Comment:
This was fiddly to work with; not confident about allowing automatic updates with `^1.8.0`

Reply:
Agreed with @robertparkinson to leave as is for now